birdhouse/components/weaver/default.env

```diff
@@ -96,6 +99,17 @@ export WEAVER_WPS_OUTPUTS_PATH="/wpsoutputs/weaver"
 export WEAVER_WPS_OUTPUTS_DIR='${BIRDHOUSE_WPS_OUTPUTS_DIR}/weaver'
 export WEAVER_WPS_WORKDIR="/tmp/wps_workdir/weaver"
 
+# Optional alternate endpoint that will redirect to Weaver.
+# If explicitly set to empty value, it will not be configured in the proxy.
+export WEAVER_ALT_PREFIX=/ogcapi
+export WEAVER_ALT_PREFIX_PROXY_LOCATION='
+    $([ -z "${WEAVER_ALT_PREFIX}" ] && echo "" || echo "
+    location ~ ^${WEAVER_ALT_PREFIX}(.*)\$ {
+        return 302 /${WEAVER_MANAGER_NAME}\$1;
```

Comment:
Is there any difference technically? I think it's just semantic between permanently vs temporarily? Since all endpoints can be configured, they seem more like temporary. Seems like all other components also used 302.

Reply:
- It speeds things up for the client since it now knows to request the redirected URL directly. 
- Endpoints are not going to be changed regularly enough for it to make sense to treat them as temporary
- All others should probably use 301 as well

Reply:
changed